.github/workflows/iroha2-dev.yml

```
@@ -1,30 +1,21 @@
-name: I2::Dev::Deploy
+name: I2::Dev::Publish
 
 on:
   push:
     branches: [iroha2-dev]
 
 env:
   CARGO_TERM_COLOR: always
-  REPOSITORY_URL: '${{ github.server_url }}/${{ github.repository }}.git'
 
 # Note jobs in this workflow are run on `push` meaning that there's
 # no point in burning our AWS self-hosted runners' time. Hence
 # `ubuntu-latest` and not `[self-hosted, Linux]`.
 jobs:
-  deploy:
+  publish_image:
```

```suggestion
  dockerhub:
```

`I2::Dev::Publish::publish_image` => `I2::Dev::Publish::dockerhub`

This will become even more relevant once we start publishing binary packages to Ubuntu, `nix` and so on.